superset/Dockerfile

```
@@ -56,6 +56,11 @@ RUN python3 -m venv /stackable/app \
         pydruid \
         python-ldap \
         trino[sqlalchemy] \
+        # Enables OIDC with superset. Since https://github.com/stackabletech/superset-operator/pull/530
+        # user can inject arbitrary python code into superset_conf.py. Thus OIDC configs can be changed
+        # and we want to offer necessary libraries
+        Flask_OIDC==2.2.0 \
+        Flask-OpenID==1.3.1\
```

If I understand it correctly, this doesn't enable OIDC, but adds dependencies for anyone who wants to opt-in, right?

If so, the comment needs revising. Maybe something like:

```suggestion
        # Add optional dependencies for use in custom Superset configurations.
        # Since https://github.com/stackabletech/superset-operator/pull/530
        # users can add custom configuration to superset_conf.py.
        Flask_OIDC==2.2.0 \
        Flask-OpenID==1.3.1\
```